README.md

```diff
@@ -76,7 +76,7 @@ please share it by [posting a link](https://github.com/willfaught/paige/discussi
 
 ## Setup
 
-1. [Install Hugo](https://gohugo.io/installation/) (the extended version, and at least 0.111.3).
+1. [Install Hugo](https://gohugo.io/installation/) (the extended version, and at least 0.123.0).
```

Comment:
The original version is correct.

Reply:
> The original version is correct.

Not sure about that, see my comment [below](https://github.com/willfaught/paige/pull/69#issuecomment-2159857318).